parser/parser.go

```diff
@@ -1063,7 +1067,12 @@ func (p *parser) parseDirective(ctx *context, g *TokenGroup) (*ast.DirectiveNode
 
 func (p *parser) parseComment(ctx *context) (ast.Node, error) {
 	cm := p.parseHeadComment(ctx)
-	node, err := p.parseToken(ctx, ctx.currentToken())
+	nextTok := ctx.currentToken()
```

Comment:
Please use this snippet and don't use `nextTok` as the variable name.
```
if tk := ctx.currentToken(); tk == nil {
	return cm, nil
}
```

Reply:
Ah, Sorry, please use `ctx.isTokenNotFound()` for null checking, so this codes will become like following.

```
if ctx.isTokenNotFound() {
  return cm, nil
}
```